stores-api/epoch-api/src/main/java/com/bloxbean/cardano/yaci/store/api/epoch/controller/ParamProposalController.java

```diff
@@ -12,14 +14,17 @@
 
 import java.util.List;
 
+@Slf4j
 @RestController
-@RequestMapping("${apiPrefix}/epoch/param-propsals")
 @RequiredArgsConstructor
-@Slf4j
+@Tag(name = "Network Service")
+@RequestMapping("${apiPrefix}/network")
```

Comment:
Does other provider group param proposals under  "Network" service? But it makes sense .

Reply:
Yes, In Koios it is under Network Service